packages/react-badge/src/components/PresenceBadge/usePresenceBadgeStyles.ts

```diff
@@ -0,0 +1,112 @@
+import { ax, makeStyles } from '@fluentui/react-make-styles';
+import { PresenceBadgeState } from './PresenceBadge.types';
+import { useRootStyles } from '../../Badge';
+
+export const usePresenceBadgeRootStyles = makeStyles<PresenceBadgeState>([
+  [
+    null,
+    {
+      padding: 0,
+      justifyContent: 'center',
+      alignItems: 'center',
+    },
+  ],
+  [
+    s => s.status === 'busy',
+    theme => ({
+      backgroundColor: theme.global.palette.cranberry.primary,
+      borderColor: theme.global.palette.cranberry.primary,
+    }),
+  ],
+  [
+    s => s.status === 'away',
+    theme => ({
+      backgroundColor: theme.global.palette.yellow.primary,
+      borderColor: theme.global.palette.yellow.primary,
+    }),
+  ],
+  [
+    s => s.status === 'available',
+    theme => ({
+      backgroundColor: theme.global.palette.lightGreen.primary,
+      borderColor: theme.global.palette.lightGreen.primary,
+    }),
+  ],
+  [
+    s => s.status === 'offline',
+    theme => ({
+      backgroundColor: 'transparent',
+      color: theme.global.palette.grey[38],
+      borderColor: theme.global.palette.grey[38],
+    }),
+  ],
+  [
+    s => s.status === 'oof',
+    theme => ({
+      backgroundColor: 'transparent',
+      color: theme.global.palette.magenta.primary,
+      borderColor: theme.global.palette.magenta.primary,
+    }),
+  ],
+  [
+    s => !s.inOffice,
+    {
+      background: 'transparent',
+    },
+  ],
+  [
+    s => !s.inOffice && s.status === 'available',
+    theme => ({
+      color: theme.global.palette.lightGreen.primary,
+    }),
+  ],
+  [
+    s => !s.inOffice && s.status === 'busy',
+    theme => ({
+      color: theme.global.palette.red.primary,
+    }),
+  ],
+  [
+    s => !s.inOffice && s.status === 'away',
+    theme => ({
+      color: theme.global.palette.yellow.primary,
+    }),
+  ],
+  [
+    s => s.size === 'small',
+    {
+      width: '16px',
+      height: '16px',
+    },
+  ],
```

Comment:
Ditto, aren't the sizes handled by `useBadgeStyles`?

Reply:
Not really, there we have only `minWidth` that generated bigger width with current  font-sizes.

Reply:
So is the bug that the font sizes are wrong? It seems like any Badge with just an icon should be a circle/square, regardless of whether it's a Badge or PresenceBadge, right?